torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -347,12 +353,16 @@ const RegisterOperators MKLDNNHardSwishOpReg({
             false),
         AliasAnalysisKind::FROM_SCHEMA),
     torch::jit::Operator(
-        "prim::MKLDNNRelu6(Tensor(a!) self) -> Tensor(a!)",
-        createUnaryOp(
-            [](at::Tensor output, at::Tensor input) {
-              at::cpu::hardtanh_out(output, input, 0.f, 6.f);
-            },
-            false),
+        "prim::MKLDNNHardTanh(Tensor self) -> Tensor",
+        [](const Node* n) -> Operation {
+          auto min_val = n->f(attr::min_val);
```

Comment:
can we keep the same schema as original `hardtanh`? and only fuse under the invariant that `min` and `max` are constants

Reply:
Resolved